.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+on: [push]
+jobs:
+  lint-test:
+    name: Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    matrix:
+      ruby:
+        - 2.4
+        - 2.5
+        - 2.6
+        - 2.7
```

Comment:
Out of curiosity: 
- Does this usage of `matrix` mean you'll end up generating four `lint-test` jobs - one per version of ruby listed here? 
- If so, why would you want that? Why not just run tests with whatever single version of ruby is this gem will be used with in the rails app (or wherever)?

Reply:
This is a public gem, there are others who might be using it on a different ruby version.

Reply:
Yeah, what @nguyenj said. If you check the `.travis.yml` file I deleted in this PR, you'll find it tested on ruby 2.1, 2.2, and 2.3. So I've updated the ruby version to all versions supported by https://github.com/actions/setup-ruby.